libs/net/services/Helpers/IngestManagerFactory`.cs

```diff
@@ -35,7 +35,7 @@ public IngestManagerFactory(IServiceProvider serviceProvider)
     /// </summary>
     /// <param name="ingest"></param>
```

Comment:
Missing parameter in docs.  If you do a `dotnet build` it should show you a number of linting issues.